recipe/meta.yaml

```
@@ -10,15 +10,16 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38]
```

```suggestion
  skip: True  # [py<37]
```
https://github.com/python-hyper/wsproto/blob/1.2.0/setup.py#L42